scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -72,48 +72,50 @@ private val TIMEOUT_ERROR_REGEX = Pattern.compile(
 )
 
 /**
- * Generate a summary from the given raw ScanCode [result], using [startTime] and [endTime] metadata. From the
- * [scanPath] the package verification code is generated. If [parseExpressions] is true, license findings are preferably
- * parsed as license expressions.
+ * Generate a summary from the given raw ScanCode [result]. From the [scanPath] the package verification code is
+ * generated. If [parseExpressions] is true, license findings are preferably parsed as license expressions.
```

Comment:
Please provide a rationale why it makes sense to align with the clearly defined implementation.

I believe using the system time is less error prone and simpler.

Reply:
> I believe using the system time is less error prone and simpler.

But it's additional code instead of making reuse of existing code. I've amended the commit message, which now also refers to 2d0038d39a39606f6163a60f4e82ac9116c08e95 for some background information.